apps/labs/posts/conda-on-colaboratory.md

```diff
@@ -0,0 +1,109 @@
+category: [Packaging]
+featuredImage:
+  src: /posts/conda-on-colaboratory/blog_feature_surbhi.svg
+  alt: 'Condacolab logo with the assigned mentors and intern.'
+hero:
+  imageSrc: /posts/conda-on-colaboratory/blog_hero_surbhi.svg
+  imageAlt: 'Condacolab logo.'
+---
+
+## Introduction
+
+Hi, I am [Surbhi](https://github.com/ssurbhi560), and this blog post is about the work I did over the last three most exciting months in my internship at Quansight Labs. My mentor during the internship was the very amazing [Jaime Rodríguez-Guerra](https://github.com/jaimergp). The project I worked on during the internship was [condacolab](https://github.com/conda-incubator/condacolab). We will shortly go into what `condacolab` is, but before that, let's understand a little about the two familiar names that we see here, which are **conda** and **colab** and also why we needed `condacolab` in the first place.
+
+## Why condacolab? 🐍
+
+[Colab](https://colab.research.google.com/) is a product from Google Research and is widely used by software developers and data scientists today. Some reasons behind its success are:
+
+* It is cloud-based.
+* It comes preinstalled with just about every library you could need in the cloud.
+* It is also not limited by the computer's available RAM, disk space, GPU, and CPU. 
+
+However, some users might find that Colab has some limitations in some areas, like:
+
+* Colab is still using and locked to Python 3.7. So, if you need to use any other Python version in Colab, you won't be able to do that easily. 
+* Colab ships many preinstalled packages, but users can only use `apt` or `pip` to update or obtain new ones.
+
+This means you won't be able to use any other package manager like [conda](https://docs.conda.io/projects/conda/en/latest/), which can be a better option for certain projects and data-science fields. You might be able to do get `conda` on colab after some hacking, but this kind of workarounds tend to be break often and lack reusability.
+
+To help folks with these limitations and make Colab more user-friendly for `conda` users, we have condacolab.
+
+## What is condacolab, and how does it work?
+
+[Condacolab]((https://github.com/conda-incubator/condacolab)) is a tool that lets you deploy a Miniconda installation easily on Google Colab notebooks and use conda or mamba in colab.
+
+---
+
+![Condacolab's previous installation](/posts/conda-on-colaboratory/previous_condacolab_installation.png)
+
+---
+
+The way it worked before was by installing the Miniconda distribution on top of the system's Python at `/usr/local`, and then adding a few configuration files to ensure we stay with Python 3.7. Finally, we wrapped the Python executable to redirect and inject some environment variables needed to load the new libraries installed by `conda`. Since we need to re-read `LD_LIBRARY_PATH`, we issued a Jupyter kernel restart.
```

Comment:
```suggestion
The way condacolab worked before was by installing the Miniconda distribution on top of the system's Python at `/usr/local`, and then adding a few configuration files to ensure we stay with Python 3.7. Finally, it wrapped the Python executable to redirect and inject some environment variables needed to load the new libraries installed by `conda`. Since we need to re-read `LD_LIBRARY_PATH`, it triggered a Jupyter kernel restart.
```